README.md

```diff
@@ -1,24 +1,24 @@
-@asymmetrik/fhir-server-mongo
-=============================
+`@asymmetrik/node-fhir-server-mongo` [![Build Status](https://travis-ci.org/Asymmetrik/node-fhir-server-mongo.svg?branch=develop)](https://travis-ci.org/Asymmetrik/node-fhir-server-mongo)
+====================================
 
 ## Intro
 
 ## Dependencies
-> This project requires nodejs and mongo to be installed on your system.
+> This project requires docker for local development. You can deploy it with or without docker. To deploy it separately, you need a server with the latest LTS for [Node.js](https://nodejs.org/en/) installed and a mongo database that can be reached by the node server.
 
-1. Install the latest LTS for [Node.js](https://nodejs.org/en/).
-2. Install the latest MongoDB Community Edition [https://www.mongodb.com/download-center?jmp=nav#community](https://www.mongodb.com/download-center?jmp=nav#community).
+1. Install the latest [Docker Community Edition](https://www.docker.com/community-edition) for your OS.
 
 ## Getting Started
+1. Run `docker-compose up`.
 
-There are some temporary steps to do before developing until this is published on Github and npm. You must currently clone https://bitbucket.org/asymmetrik/asy-fhir-server and run `npm link` inside that directory. Once you do, return to this directory, and link-install it with `npm link @asymmetrik/fhir-server-core`.
+## Usage
+There are several other commands that may be useful. You can run all of them through `docker-compose run <service>` or `docker-compose exec <service>`. Use `docker-compose exec` if you already have the app running, otherwise use `docker-compose run` to run it once. The available scripts are:
 
-1. Run `npm install`.  If you prefer yarn, you can also run `yarn`.
-2. Start your mongo server.
-3. Start the application with `yarn start` or `npm start`.
+* `docker-compose exec fhir test` - lint the src directory and use jest to run all of our tests
+* `docker-compose exec fhir test:lint` - only lint your src directory
+* `docker-compose exec fhir test:jest` - only use jest to run all the tests
 
-## Usage
-Coming soon...
+The two other npm scripts available are `start` and `nodemon`. `nodemon` is used when you run `docker-compose up` and if you ever deploy, you should use the `start` command.
```

Comment:
I'd clean up this sentence to be something like

> The other two npm scripts available are:
* `start` - runs the server in production mode
* `nodemon` - runs the server in development mode, and is the default when you run `docker-compose up`

Reply:
Also, it's not `npm`, it's `yarn`, right?  And if you're running them from Docker you actually don't care which it it.  Just remove the word `npm` from the sentence.  :)